G/gdk_pixbuf/build_tarballs.jl

```diff
@@ -49,19 +66,23 @@ products = [
 # Some dependencies are needed only on Linux and FreeBSD
 linux_freebsd = filter(p->Sys.islinux(p)||Sys.isfreebsd(p), platforms)
 
+# gobject_introspection is needed only on x86_64-linux-gnu
+introspect_platform = filter(p -> triplet(p) == "x86_64-linux-gnu", platforms)
+
 # Dependencies that must be installed before this package can be built
 dependencies = [
     # Need a host gettext for msgfmt
     HostBuildDependency("Gettext_jll"),
     # Need a host glib for glib-compile-resources
     HostBuildDependency("Glib_jll"),
-    Dependency("Glib_jll"; compat="2.68.1"),
+    Dependency("Glib_jll"; compat="2.68.3"),
```

Comment:
Is this change necessary?

Reply:
That GLib version was set in the build for gobject_introspection (by me, oops) and I'm seeing:
```
ERROR: LoadError: Unsatisfiable requirements detected for package Glib_jll [7746bdde]:
 Glib_jll [7746bdde] log:
 ├─possible versions are: 2.59.0-2.68.3 or uninstalled
 ├─restricted to versions 2.68.1 by an explicit requirement, leaving only versions 2.68.1
 └─restricted by compatibility requirements with gobject_introspection_jll [6306888c] to versions: 2.68.3 — no versions left
   └─gobject_introspection_jll [6306888c] log:
     ├─possible versions are: 1.72.0 or uninstalled
     └─restricted to versions * by an explicit requirement, leaving only versions 1.72.0
```
There's no reason to require 2.68.3 in gobject_introspection_jll -- could we create a new version of that with 2.68.1?

Reply:
Uhm, I see.  No, ok, that should be fine, I just wanted to understand what's going on.